pkg/common/args_test.go

```diff
@@ -76,14 +77,8 @@ func TestValidateArgs(t *testing.T) {
 
 			// Call the function under test
 			ValidateArgs()
-			if viper.GetString("skip") != tc.expectedSkip {
-				t.Errorf("expected skip to be [%s], got [%s]", tc.expectedSkip, viper.GetString("skip"))
-			}
-			if viper.GetString("focus") != tc.expectedFocus {
-				t.Errorf("expected focus to be [%s], got [%s]", tc.expectedFocus, viper.GetString("focus"))
-			}
```

Comment:
This is a good check to have to test that https://github.com/kubernetes-sigs/hydrophone/blob/main/pkg/common/args.go#L52 is working properly.

Reply:
my bad, removed it by mistake. let me add it back